src/gui/mainwindow.cpp

```diff
@@ -152,10 +152,10 @@ MainWindow::MainWindow(QWidget *parent)
 #ifndef Q_OS_MACOS
 #ifdef Q_OS_UNIX
     const QIcon appLogo = Preferences::instance()->useSystemIconTheme()
-        ? QIcon::fromTheme("qbittorrent", QIcon(":/icons/qbittorrent-tray.svg"))
-        : QIcon(":/icons/qbittorrent-tray.svg");
+        ? QIcon::fromTheme("qbittorrent", UIThemeManager::instance()->getIcon(QLatin1String("qbittorrent-tray")))
+        : UIThemeManager::instance()->getIcon(QLatin1String("qbittorrent-tray"));
```

Comment:
Isn't this logic already in UIThemeManager?

Reply:
a call like this 
```cpp
    const QIcon appLogo(UIThemeManager::instance()->getIcon(QLatin1String("qbittorrent"), QLatin1String("qbittorrent-tray")));
```
is not equal to the original code.

Reply:
Seems we just need "qbittorrent.svg" icon (e.g. as copy of "qbittorrent-tray.svg").
It's a good idea in itself to have main app icon having the same name.

Reply:
>Seems we just need "qbittorrent.svg" icon (e.g. as copy of "qbittorrent-tray.svg").
It's a good idea in itself to have main app icon having the same name.

I don't understand, 

Can we just leave it for another time, I don't want to change the logic here

Reply:
>is not equal to the original code

BTW, although it is not equal to the original code it seems to give the same result.
>Can we just leave it for another time, I don't want to change the logic here

Now this code should be refactored, since such logic is unacceptable outside of the UIThemeManager class. Why not finish this job here?